src/wilcoxon.jl

```diff
@@ -163,6 +163,26 @@ function StatsAPI.pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
+function psignrank(W::T, n::T, lower_tail::Bool) where {T<:Integer}
+    max_W =  n * (n + one(T)) ÷ (one(T) + one(T))
```

Comment:
`>>` should work even with generic integer types (but IMO we shouldn't be worried about them at this point anyway and they are not tested at all):
```suggestion
    max_W =  (n * (n + 1)) >> 1
```